src/main/java/Duke.java

```diff
@@ -1,10 +1,368 @@
+import java.io.*;
+import java.util.Scanner;
+import java.time.*;
+import java.time.format.DateTimeFormatter;
+
+// level 9 commits branch
+// sample level 9 change
+
 public class Duke {
+
+    public static int counter = 0; // this has to be a global variable
+    public static Task[] tasks = new Task[100]; // another global variable
+
+    /*
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        }
+        catch (DukeException e) {
+            ui.showLoadingError();
+            tasks = new TaskList();
+        }
+    }
+
+    public void run() {
+        ui.greetingMessage();
+        String input = ui.scanInput();
+
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+        new Duke("").run();
+    }
+
+    */
+    public static void main(String[] args) {
+
+        try {
+            loadFile(); // load the file first
+        }
+        catch (FileNotFoundException exception) {
+            System.out.println("File not found!");
+        }
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+
+        processRequests(); // process all user requests
+
+        updateFile(); //then need to write back the new list to the file
```

Comment:
Maybe you want to updateFile() in methods like delete, add or mark completed? In the case where duke crashes, it will still save the previous requests.